### PR TITLE
Add global controls for dimmer, white, and amber

### PR DESCRIPTION
**Pull request by somaholiday**
Removed some older code related to fixture types we aren't using.

Revised the existing white/amber controls to work with our current output.

Added a dimmer control that sets the global brightness by fixture type.

Commented out the `Output` code that was conflicting with our current output and causing the flickering.

### CODE REVIEW

File: src/main/java/org/yomigae/UIDimmer.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  public String filename;
+
+  public int parDimmer = 255;
+  public int wallWasherDimmer = 255;
+
+  public UIDimmer(final LXStudio.UI ui, String title, String filenameBase) {
+    super(ui, title, filenameBase + ".json");
+    filename = filenameBase + ".json";
+
+    parDimmer = (int)registerCompoundParameter(PAR_DIMMER, 255f, 0f, 255f).getValuef();
```
Comment (somaholiday) [inline src/main/java/org/yomigae/UIDimmer.java:19]: I went back and forth on whether this should just be a discrete parameter.  It probably should.

File: src/main/java/org/yomigae/Yomigae.java [hunk 2]
```diff
@@ -149,19 +146,15 @@ public void setup() {
...
-    prodWashWhiteControl = (UIWhiteControl) new UIWhiteControl(lx.ui, "PRODWASH WHITE", "prodwash_whites")
+    parWhiteControl = (UIWhiteControl) new UIWhiteControl(lx.ui, "WHITE : PAR", "white_par")
         .setExpanded(false).addToContainer(lx.ui.leftPane.global);
-    oppSkWhiteControl = (UIWhiteControl) new UIWhiteControl(lx.ui, "OPPSK WHITE", "oppskpar_whites")
+    wallWasherWhiteControl = (UIWhiteControl) new UIWhiteControl(lx.ui, "WHITE : WALL WASHER", "white_wallwasher")
         .setExpanded(false).addToContainer(lx.ui.leftPane.global);
 
     if (enableOutput) {
-      Output.configureE131Output(lx, Output.LightType.OPPSKPAR);
+      // Output.configureE131Output(lx, Output.LightType.PRODPAR);
```
Comment (somaholiday) [inline src/main/java/org/yomigae/Yomigae.java:157]: The magic line that fixed our flickering 😬

File: src/main/java/org/yomigae/UIFixtureType.java [hunk 3]
```diff
@@ -15,9 +15,9 @@
   public UIFixtureType(final LXStudio.UI ui, LX lx) {
     super(ui, title, filename);
     this.lx = lx;
-    registerDiscreteParameter("Type", 1, 1, Output.LightType.OPPSKPAR.getValue() + 1);
+    registerDiscreteParameter("Type", 1, 1, Output.LightType.PRODPAR.getValue() + 1);
```
Comment (hdanak) [inline src/main/java/org/yomigae/UIFixtureType.java:18]: What is the purpose of choosing fixture type?

File: src/main/java/org/yomigae/UIFixtureType.java [hunk 4]
```diff
@@ -15,9 +15,9 @@
...
     // TODO(tracy): UIToggleSet is crashing for some reason, just use a discrete knob for now.
-    String[] toggleOptions = {"LaLuce", "Prod Par", "Prod Wash"};
+    String[] toggleOptions = {"Prod Par", "Prod Wash"};
```
Comment (hdanak) [inline src/main/java/org/yomigae/UIFixtureType.java:20]: This doesn't do anything right?

File: src/main/java/org/yomigae/UIFixtureType.java (hunk 3 above)
Comment (somaholiday) [reply src/main/java/org/yomigae/UIFixtureType.java:18]: This was used by Tracy's previous Output layer.  I think I only made this change to get everything to compile.

File: src/main/java/org/yomigae/UIFixtureType.java (hunk 4 above)
Comment (somaholiday) [reply src/main/java/org/yomigae/UIFixtureType.java:20]: Yup, we're not using it in our Output layer.

**Pull request merged**